hello-world/README.md

```diff
@@ -1,40 +1,73 @@
+3. Enter the secret key you created in Step 1 into the Initial Value column of the user-defined variable.
+
+    ![alt text](https://learn.akamai.com/en-us/webhelp/edgeworkers/edgeworkers-user-guide/GUID-ABA87948-098E-4571-A001-7BC6F3E20381-low.png "Setting Property Variables")
+   * You'll also re-use this secret key, when using the [EdgeWorkers CLI](https://developer.akamai.com/cli/packages/edgeworkers.html) to generate the authentication token for debugging.
+4. Add blank Rule to delivery property called **Hello World**
+5. Add match condition for path /hello-world
+6. Add EdgeWorkers behaviour
+7. Save the property
+8. Create new EdgeWorker Identifier by clicking **EdgeWorkers Management application** in the behaviour note
+9. Click the **Create Worker ID** button
+10. Enter **Hello World** in the Name field 
+11. Select the group you want the EdgeWorker to be available in
+12. Click the **Create Worker ID** button
+13. Click the newly created **ID** or **Name**
+14. Click **Create Version** button
+15. Drag and Drop or Select the hello-world.tgz file 
+16. Select the **Create Version** button.
+17. Active the newly added version to staging/production from the action menu
+18. Reload the property, select the newly created EdgeWorker
+19. Save the property
+20. Active the property to staging/production 
+21. Use the secret key you created in Step 1 to generate an authentication token using this [EdgeWorkers CLI](https://developer.akamai.com/cli/packages/edgeworkers.html) command.   
+    ```
+    $ akamai edgeworkers auth fef77a483a6dd85b45a6c5092f1c178a6eaf21e56a3b69195e33f53070eec669
+      ---------------------------------------------------------------------------------------------------------------------------   
+      Add the following request header to your requests to get additional trace information.
+      Akamai-EW-Trace: st=1603897978~exp=1603899778~acl=/*~hmac=090513d88251d13ceae6dd4d35504498f1ea59c9d081fe8f86ffcf01361cf53f
+      ---------------------------------------------------------------------------------------------------------------------------```
+22. To have debug and logging response headers returned add two headers to the request. See [Enhanced debug headers](https://learn.akamai.com/en-us/webhelp/edgeworkers/edgeworkers-user-guide/GUID-F888493F-6186-4400-89B4-0AEDF872DFC9.html) for more information.
+    * a Pragma header with a value of akamai-x-ew-debug `Pragma: akamai-x-ew-debug`
+    * The Akamai-EW-Trace header created in Step 21 `Akamai-EW-Trace: st=1603897978~exp=1603899778~acl=/*~hmac=090513d88251d13ceae6dd4d35504498f1ea59c9d081fe8f86ffcf01361cf53f`
+## Example
 
     GET /hello-world
     Host: mysite
-    
-    
-    <html><body><h1>Hello World From Akamai EdgeWorkers</h1></body></html>
+    Pragma: akamai-x-ew-debug
+    Akamai-EW-Trace: st=1603897978~exp=1603899778~acl=/*~hmac=090513d88251d13ceae6dd4d35504498f1ea59c9d081fe8f86ffcf01361cf53f
```

Comment:
Should we stub out the hmac value since it's not going to be valid for anyone besides you?

Reply:
it's valid for the key included in the readme and will expire. So up to you.